### PR TITLE
Make script compatible to python 2.7 and python 3.6

### PR DESCRIPTION
**Pull request by vteratipally**
Signed-off-by: Varsha Teratipally teratipally@google.com

### CODE REVIEW

File: cluster/gce/configure.sh [hunk 1]
```diff
@@ -55,7 +71,11 @@ fetch_env() {
     # Convert the yaml format file into a shell-style file.
     eval $(python -c '''
```
Comment (bsdnet) [inline cluster/gce/configure.sh:72]: s/python/${PYTHON}/g

**Comment by bsdnet** [comment]
/assign @Random-Liu  @dims 

This blocks https://github.com/kubernetes/test-infra/pull/19207
and similar to https://github.com/containerd/containerd/pull/4559

**Comment by vteratipally** [comment]
I have tested this by creating a gce instance with config and the containerd and docker services are running properly now.

**Comment by Random-Liu** [comment]
/lgtm
/approve

This is a GCE specific change that unblocks the OSS Kubernetes testing.

**Comment by bsdnet** [comment]
/lgtm

One nit: Can you merge two commits into one in this PR?

**Comment by estesp** [comment]
The commits have no `Signed-off-by: ` and therefore the DCO check is failing the CI run

**Pull request closed without merging**